README.md

```diff
@@ -1,75 +1,53 @@
 # Phoxygen Website
 
-This is the official Phoxygen website.
+This is the official Phoxygen website, to be published at http://www.phoxygen.com/.
 
-We started from the Jekyll implementation of the [Stylish
-Portfolio](http://startbootstrap.com/template-overviews/stylish-portfolio/)
-template by [Start Bootstrap](http://startbootstrap.com/).
+Template based on [Stylish
+Portfolio](http://startbootstrap.com/template-overviews/stylish-portfolio/).
 
-See the site in action at https://www.phoxygen.com/
+## Developer Workflow
 
-## Dev Workflow
-
-The website can be accessed on http://localhost:4000/
-
-You can either install the dependencies manually or use the Docker recipe.
+Jekyll is required to generate this website.
+It can be either installed manually or run from a Docker container.
 
 ### Manual Install
 
-Prerequisite:
-
-- Ruby 2.1.0 or later
-- bundler
+Prerequisite: Ruby 2.1.0 or later
 
 ```bash
 gem install bundler
 bundle install
-# jekyll serve
-bundle exec jekyll serve --watch
+bundle exec jekyll serve
 ```
 
-### With Docker
-
-You can let docker deal with the ruby gems.
-
-**/!\ Make sure your Docker client points to a local Docker daemon.**
-These steps are not suitable for a remote docker instance.
+Note: `jekyll serve` should be equivalent to the last line but might raise
+a dependency issue (e.g. with kramdown).
```

Comment:
The official doc of bundler actually clarify that:

 > In some cases, running executables without bundle exec may work, if the executable happens to be installed in your system and does not pull in any gems that conflict with your bundle.
>
> However, this is unreliable and is the source of considerable pain. Even if it looks like it works, it may not work in the future or on another machine.

That's probably why it was working on my system and not on yours last time. So the correct command is clearly  `bundle exec jekyll serve` :-)